extensions-contrib/compressed-bigdecimal/src/main/java/org/apache/druid/compressedbigdecimal/CompressedBigDecimalBufferAggregator.java

```diff
@@ -71,10 +71,11 @@ public void init(ByteBuffer buf, int position)
   @Override
   public void aggregate(ByteBuffer buf, int position)
   {
-    CompressedBigDecimal<?> addend = selector.getObject();
+    CompressedBigDecimal addend = Utils.objToCompressedBigDecimal(selector.getObject());
     if (addend != null) {
       Utils.accumulate(buf, position, size, scale, addend);
```

Comment:
Just double checking, this `Utils.accumulate()` will scale the scale right?

Reply:
yes, i moved it there because 
1. we have access to scale to pass in
2. accumulate will always be functionally correct

the only reason to move it elsewhere is if we somehow were going to use a CBD more than once in a new scale to avoid repeatedly changing the scale. that doesn't seem the case in druid